datascience/tables.py

```diff
@@ -243,26 +243,29 @@ def values(self, label):
 
         ``__getitem__`` is aliased to this, so bracket notation can be used:
 
+        >>> letter = ['a', 'b', 'c', 'z']
+        >>> count = [9, 3, 3, 1]
+        >>> points = [1, 2, 2, 10]
+        >>> t = Table([letter, count, points], ['letter', 'count', 'points'])
         >>> t.values('letter') == t['letter']
-        True
+        array([ True,  True,  True,  True], dtype=bool)
```

Comment:
Actually, let's not have this in the doctests but instead just write it in the description for the method. Something like: `t.values('letter')` is equivalent to `t['letter']`.